src/app/modules/reports/components/time-entries-table/time-entries-table.component.spec.ts

```diff
@@ -76,6 +79,24 @@ describe('Reports Page', () => {
       expect(component.dtTrigger.next).toHaveBeenCalled();
     });
 
```

Comment:
It would be useful, if we add another test, to test only the `isValidUri` function. This test could be parametrized and could use the following values:
```
url = 'http://example.com' ; expected_value = true
url = 'https://example.com' ; expected_value = true
url = 'no-url-example' ; expected_value = false
```